wpiutil/.styleguide

```diff
@@ -70,6 +70,7 @@ generatedFileExclude {
   src/main/native/resources/
   src/test/native/cpp/UnitsTest\.cpp$
   src/test/native/cpp/llvm/
+  thirdparty/
```

Comment:
I think the correct place to put this is not .styleguide, but rather add it to the root .gitignore (which is also read by wpiformat, and also addresses git looking at these files).